src/pages/resultsView/oncoprint/AddClinicalTracks.tsx

```diff
@@ -87,24 +87,26 @@ export default class AddClinicalTracks extends React.Component<
     private clear(clinicalAttributeIds: string[]) {
         this.props.onChangeSelectedClinicalTracks(
             _.difference(
-                this.props.getSelectedClinicalAttributeIds(),
-                clinicalAttributeIds
+                this.props.getSelectedClinicalAttributes(),
```

Comment:
i would not expect that that this would work because any new instance of ClinicalTrackConfig will always evaluate as not equal to existing instances. They will always be different values even if they represent the same data.